config/prod.exs

```diff
@@ -204,7 +204,9 @@ config :archethic, Archethic.SharedSecrets.NodeRenewalScheduler,
 config :archethic, Archethic.SelfRepair.Scheduler,
   # Every day at 00:05:00
   # To give time for the beacon chain to produce summary
-  interval: System.get_env("ARCHETHIC_SELF_REPAIR_SCHEDULER_INTRERVAL", "0 5 0 * * * *")
+  interval: System.get_env("ARCHETHIC_SELF_REPAIR_SCHEDULER_INTRERVAL", "0 5 0 * * * *"),
+  # Availability application date 10 minutes after self repair
+  availability_application: 900
```

Comment:
I don't get those numbers
- 5s -> 10s
- 10m -> 900s

If you want to add 10m to the summary time, it should be 600s, no ?